packages/docs/docs/tutorial/part4.md

```diff
@@ -0,0 +1,110 @@
+---
+sidebar_position: 4
+---
+
+# Tutorial Part 4 - Document Q&A
+
+One of the most powerful capabilities of Large Language Models is the ability to
+answer questions about the contents of a set of documents. AI. JSX provides a powerful
+component called `<DocsQA>` that can answer a question about a corpus of documents
+using the LLM.
+
+Here's an example of how this will look in your app:
+
+```tsx filename="packages/tutorial/src/part4.tsx"
+<DocsQA question="When did the storm occur?" corpus={corpus} chunkLimit={5} chunkFormatter={GetChunk} />
+```
+
+## How does Documents Q&A work?
```

Comment:
I feel like there's a fair bit of overlap here with https://docs.ai-jsx.com/guides/docsqa – can we consolidate?